docs/altInstances.md

```diff
@@ -24,6 +24,27 @@ class MyAlt extends Alt {
 var flux = new MyAlt();
 ```
 
+You can then pass your flux instance using withAltContext.
```

Comment:
You can set your entire app context by wrapping your root component with `withAltContext`.